.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python:
+          - { VERSION: "2.7", TOXENV: "py27", ALLOW_FAILURE: false }
+          - { VERSION: "2.7", TOXENV: "py27-flake8", ALLOW_FAILURE: false}
+          - { VERSION: "3.5", TOXENV: "py35", ALLOW_FAILURE: false }
+          - { VERSION: "3.6", TOXENV: "py36", ALLOW_FAILURE: false }
+          - { VERSION: "3.7", TOXENV: "py37", ALLOW_FAILURE: false }
+          - { VERSION: "3.8", TOXENV: "py38-flake8,readme,doclint,docs,commitlint",ALLOW_FAILURE: false }
+          - { VERSION: "3.8", TOXENV: "docstrings", ALLOW_FAILURE: true}
+          - { VERSION: "3.8", TOXENV: "notebooks", ALLOW_FAILURE: true}
+          - { VERSION: "pypy2", TOXENV: "pypy", ALLOW_FAILURE: false }
+          - { VERSION: "pypy3", TOXENV: "pypy", ALLOW_FAILURE: false }
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.PYTHON.VERSION }}
+      - name: Install Tox and any other packages
+        run: pip install tox
+      - name: Run Tox
+        run: tox
+        env:
+          TOXENV: ${{ matrix.PYTHON.TOXENV }}
```

Comment:
We also want `GH_RECORD_MODE` here:

https://github.com/esacteksab/github3.py/blob/d7d9ace14bf434fee99ef8331577f7116b24afd3/.travis.yml#L12-L14

Reply:
Apologies, but I can't tell what that does. Google and Travis' docs are letting me down. Any idea what this was intended to do so I can find the equivalent capability with Github actions?

Reply:
It's used purely by this test suite. It's a custom environment variable that ensures we never talk to the network

Reply:
https://github.com/sigmavirus24/github3.py/blob/253b6cdb98208e4be8f3ae19ba6fdac5163494c7/tests/conftest.py#L18